site/pages/badger-profile/index.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+import InlineSVG from 'svg-inline-react';
+import React from 'react';
+
+import styles from './style.css';
+
+import twitterSVG from './SVG/twitter.svg';
+import githubSVG from './SVG/github.svg';
+import linkedinSVG from './SVG/linked-in.svg';
+
+type Badger = {
+  firstName: string,
+  lastName: string,
+  jobTitle: string,
+  slug: string,
+  imageUrl: string,
+  about: string,
+  skills: Array<string>,
+  achievements: string,
+  influence: string,
+  twitter: string,
+  github: string,
+  linkedIn: string,
+  squarespaceId: string,
+  categories: Array<{ slug: string, name: string, order: number }>,
+};
+
+const getSocialItems = badger => {
+  const socialItems = [];
+  const socialPages = [
+    { social: 'Twitter', img: twitterSVG },
+    { social: 'Github', img: githubSVG },
+    { social: 'LinkedIn', img: linkedinSVG },
+  ];
+
+  for (const s of socialPages) {
+    if (badger[s.social.toLowerCase()]) {
+      socialItems.push(<li>
+        <a
+          href={badger[s.social.toLowerCase()]}
+          title={s.social}
+          className={styles.badgerSocialIcon}
+        >
+          <InlineSVG
+            src={s.img}
+            title={s.social}
+          />
+        </a>
+      </li>);
+    }
+  }
+  return socialItems;
+};
```

Comment:
Instead of functions like `getSocialItems` that return JSX, convert them into Components, e.g., const SocialItems = badger => … and then you can use it like <SocialItems badger={badger} />